src/main/scala/cromwell/engine/backend/jes/OperationMetadata.scala

```diff
@@ -14,11 +13,18 @@ object OperationMetadata {
       metadata.get("endTime").map(_.asInstanceOf[String].toDate))
   }
 
-  val Iso8601DateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.SSSXXX")
+  val defaultDate = DateTime.parse("0")
```

Comment:
Both are uncomfortable lies... these are used when logging the submitted, started and finished time of JES jobs.